assignments/array-methods.js

```diff
@@ -55,29 +55,62 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
-let fullName = [];
-console.log(fullName);
+
+runners.forEach(function(fullName){
+    console.log(fullName.first_name, fullName.last_name)
+  })
 
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
-let allCaps = [];
-console.log(allCaps); 
+
+let allCaps = runners.map((first_name) => {
+    return {'First Name': first_name.first_name.toUpperCase()};
+});
```

Comment:
So, what you could do here so you don't have to repeat the `first_name` variable here is use parameter destructuring.
```js
let allCaps = runners.map(({first_name}) => {
    return {'First Name': first_name.toUpperCase()};
});
```
All you have to do is wrap the parameters you want to pluck out inside of curly braces.